test/PushNotificationTests/MockRawNotification.h

```diff
@@ -0,0 +1,27 @@
+﻿#include <winrt/Windows.ApplicationModel.background.h>
+#include <winrt/Windows.Storage.Streams.h>
+#include <TestDef.h>
+
+struct MockRawNotification : winrt::implements<MockRawNotification, winrt::Windows::Networking::PushNotifications::IRawNotification, winrt::Windows::Networking::PushNotifications::IRawNotification2, winrt::Windows::Networking::PushNotifications::IRawNotification3>
+{
+    MockRawNotification() {};
+
+    winrt::Windows::Foundation::Collections::IMapView<winrt::hstring, winrt::hstring> Headers();
+    winrt::hstring ChannelId();
+    winrt::hstring Content();
```

Comment:
Sholdn't ChaneelId() and Content() methods be const?

Sigh. They're WinRT methods. No such concept :-(

Nevermind